output-timestamp/index.html

```diff
@@ -1,85 +1,74 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width">
-    <title>Output timestamp example</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Web Audio API examples: Output timestamp</title>
   </head>
 
   <body>
-    <h1>Output timestamp example</h1>
-    <button class="play">Play</button>
-    <button class="stop">Stop</button>
-    <pre></pre>
+    <h1>Web Audio API examples: Output timestamp</h1>
+    <button id="play">Play</button>
+    <button id="stop" disabled="true">Stop</button>
+    <output />
```

Comment:
Removing this line fixes a Nu HTML checker error.

Reply:
I need it. (I display text there). But I can try `<output></output>` that should please the HTML checker.

Reply:
I tested it with the HTML checker and having a start tag and a closing tag fixes 3/4 if the errors.

Reply:
Fixed.